jedi/evaluate/compiled/subprocess/__init__.py

```diff
@@ -275,6 +275,12 @@ def listen(self):
         if sys.version_info[0] > 2:
             stdout = stdout.buffer
             stdin = stdin.buffer
+        # Python 2 opens streams in text mode on Windows. Set stdout and stdin
+        # to binary mode.
+        elif sys.platform == 'win32':
+            import msvcrt
+            msvcrt.setmode(stdout.fileno(), os.O_BINARY)
+            msvcrt.setmode(stdin.fileno(), os.O_BINARY)
```

Comment:
That makes a LOT of sense. Thanks for finding it. How did you realize that was the solution?